package.json

```diff
@@ -21,6 +21,7 @@
     "@angular/platform-browser": "^5.2.0",
     "@angular/platform-browser-dynamic": "^5.2.0",
     "@angular/router": "^5.2.0",
+    "angular2-jwt": "^0.2.3",
```

Comment:
Unable to build the project. Getting the following error. Not sure if we need to include install other dependencies?
![image](https://user-images.githubusercontent.com/16127222/37618336-f3fed594-2b83-11e8-9fcd-5f2135549dd3.png)